app/views/transaction.py

```diff
@@ -621,18 +726,43 @@ def delete(self, fund_transfer_id):
          """
         try:
             fund_transfer_data = FundTransfer.query.filter(FundTransfer.id == fund_transfer_id).first()
+            if not fund_transfer_data:
+                raise FundTransferObjectNotFound
+
             db.session.delete(fund_transfer_data)
             db.session.commit()
 
             logger.info("Response for delete request for fund transfer:"
                         "Fund transfer with this id deleted successfully")
 
             return "Fund transfer record deleted successfully"
-        except Exception:
+        except FundTransferObjectNotFound:
             logger.exception("Response for delete request for fund transfer:"
                              "Fund transfer with this id does not exist")
             response = ResponseGenerator(data={},
                                          message="Fund transfer with this id does not exist",
                                          success=False,
                                          status=HTTPStatus.NOT_FOUND)
             return response.error_response()
+
+
+class MiniStatementResources(Resource):
+    def get(self):
+        try:
+            mini_statement_data = AccountTransactionDetails.query.order_by(desc(AccountTransactionDetails.id)).limit(10)
+            if not mini_statement_data:
+                raise
```

Comment:
You need to raise `MiniStatementObjectNotFound ` here.

Reply:
And pass `message` here that can be used in the ResponseGenerator.

Reply:
Made the changes as suggested.